### PR TITLE
Feat/why tlip

### PR DESCRIPTION
**Pull request by evavirseda**

### CODE REVIEW

File: src/routes/homepage/sections/whyTLIP.svelte [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
+<section class=" w-full h-full relative" {id}>
+    <div class="w-full h-full lg:flex">
+        <div class="image w-full h-full lg:w-1/2 bg-cover bg-no-repeat" />
+        <div class="bg-green-1 w-full lg:w-1/2 py-16 lg:py-28 px-6 lg:px-32 text-white">
+            <div class="mb-9">
+                <h2 class="mb-6">{HEADINGS.title}</h2>
+                <p class="">{HEADINGS.description}</p>
+            </div>
+            {#each REASONS as reason}
+                <div class="w-1/2 mb-6">
```
Comment (VmMad) [inline src/routes/homepage/sections/whyTLIP.svelte:33]: I'd do a `last:mb-0` so the last reason doesn have more space around than the first one

File: src/routes/homepage/sections/whyTLIP.svelte [hunk 2]
```diff
@@ -1,8 +1,50 @@
...
+        }
+    ];
 </script>
 
-<section class="container py-72" {id}>
-    <div>WhyTLIP</div>
+<section class=" w-full h-full relative" {id}>
+    <div class="w-full h-full lg:flex">
+        <div class="image w-full h-full lg:w-1/2 bg-cover bg-no-repeat" />
+        <div class="bg-green-1 w-full lg:w-1/2 py-16 lg:py-28 px-6 lg:px-32 text-white">
```
Comment (VmMad) [inline src/routes/homepage/sections/whyTLIP.svelte:27]: They might have changed it recently, `background-color` is now `TLIP Green-500 #005923` which is a little lighter

**Review by VmMad — COMMENTED** [review]
Great work, left some comments! :fire:

File: src/routes/homepage/sections/whyTLIP.svelte (hunk 2 above)
Comment (evavirseda) [reply src/routes/homepage/sections/whyTLIP.svelte:27]: That's right, the color palette has been updated. I'll update it and make the necessary changes :wink:

**Pull request merged**